packages/fasmifra/fasmifra.1.0.0/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+authors: "Francois Berenger"
+maintainer: "unixjunkie@sdf.org"
+homepage: "https://github.com/UnixJunkie/FASMIFRA"
+bug-reports: "https://github.com/UnixJunkie/FASMIFRA/issues"
+dev-repo: "git+https://github.com/UnixJunkie/FASMIFRA.git"
+license: "GPL-3.0-or-later"
+build: ["dune" "build" "-p" name "-j" jobs]
+install: [
+  ["cp" "bin/fasmifra_fragment.py" "%{bin}%/fasmifra_fragment.py"]
+]
+depends: [
+  "base-unix"
+  "batteries" {>= "3.3.0"}
+  "dolog" {>= "6.0.0"}
+  "dune" {>= "1.11"}
+  "minicli" {>= "5.0.2"}
```

Comment:
```suggestion
  "minicli" {>= "5.0.0"}
```
The current version is good, I will just try to relax by an epsilon the constraints